diff --git a/node_modules/@capacitor/filesystem/android/build.gradle b/node_modules/@capacitor/filesystem/android/build.gradle
index 915b0dd..cef0e70 100644
--- a/node_modules/@capacitor/filesystem/android/build.gradle
+++ b/node_modules/@capacitor/filesystem/android/build.gradle
@@ -26,7 +26,6 @@ buildscript {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
 if (System.getenv("CAP_PLUGIN_PUBLISH") == "true") {
     apply plugin: 'io.github.gradle-nexus.publish-plugin'
     apply from: file('../scripts/android/publish-root.gradle')
